app/javascript/packs/application.js

```diff
@@ -0,0 +1,355 @@
+    }
+
+    o.style = this.stylesMap[level] || level;
+    o.message = $('<div/>').text(options.message).html();
+    o.duration = this.durations[level] * 1000;
+    o.size = 'large';
+
+    $.growl(o);
+  }
+
+  titleFor (options, level) {
+    if (level === 'error') {
+      return I18n.t('js.error');
+    }
+  }
+
+  show (messages) {
+    return (() => {
+      const result = [];
+      for (var message of Array.from(messages)) {
+        const messageType = message[0];
+
+        if (this[messageType]) {
+          if ($.type(message[1]) === 'array') {
+            result.push(Array.from(message[1]).map(msg => this[message[0]](msg)));
+          } else {
+            result.push(this[message[0]](message[1]));
+          }
+        } else {
+          result.push(undefined);
+        }
+      }
+      return result;
+    })();
+  }
+}
+defaultExport.Messenger = Messenger;
+
+class Homs {
+  constructor () {
+    this.enableDateTimePicker = this.enableDateTimePicker.bind(this);
+    this.enableOrderTypePicker = this.enableOrderTypePicker.bind(this);
+    this.enableOrderAttributePicker = this.enableOrderAttributePicker.bind(this);
+    this.setAttributePickerOptions = this.setAttributePickerOptions.bind(this);
+    this.updateOrderForm = this.updateOrderForm.bind(this);
+
+    this.messenger = new Messenger();
+  }
+
+  enableDateTimePicker (allowClear) {
+    if (allowClear == null) {
+      allowClear = false;
+    }
+
+    $('.datetime-picker').each(function () {
+      const $el = $(this);
+      const format = $el.data('format') ? $el.data('format') : false;
+      const showClear = $el.data('showclear') ? $el.data('showclear') : false;
+      return $el.datetimepicker({
+        locale:    I18n.locale,
+        format,
+        showClear: showClear || allowClear
+      });
+    });
+  }
+
+  enableOrderTypePicker (allowClear) {
+    $('.order_type-picker').select2({
+      width: '100%',
+      allowClear,
+      theme: 'bootstrap',
+      formatSelection () {
+        return (node)(node.id);
+      },
+      cache:    true,
+      language: I18n.locale
+    });
+  }
+
+  enableOrderAttributePicker (allowClear) {
+    $('.order_attribute-picker').select2({
+      width: '100%',
+      allowClear,
+      theme: 'bootstrap',
+      formatSelection () {
+        return (node)(node.id);
+      },
+      cache:    true,
+      language: I18n.locale
+    });
+
+    const currentOrderType = $('.order_type-picker option:selected').val();
+
+    if (currentOrderType) {
+      this.setAttributePickerOptions(currentOrderType);
+    }
+  }
+
+  setAttributePickerOptions (orderType) {
+    return $.ajax(`/orders/order_type_attributes/${orderType}`, {
+      method:  'GET',
+      success: (data) => {
+        if (Object.keys(data.options).length > 0) {
+          const $orderAttributePicker = $('.order_attribute-picker');
+          $orderAttributePicker.empty();
+
+          $.each(data.options, (key, value) => {
```

Comment:
Why you use jQuery each?

Reply:
https://github.com/latera/homs/blob/6dd930ea5de7d1d11fab737ec7e04b05df4150b3/app/assets/javascripts/application.js.coffee#L200
I think we will get rid of jQuery, but not in this PR, it's just decaffeinating here.